app/editor/src/features/storage/constants/fileExplorerColumns.tsx

```diff
@@ -0,0 +1,155 @@
+import { IItemModel } from 'hooks/api-editor';
+import moment from 'moment';
+import {
+  FaCopy,
+  FaFileDownload,
+  FaPaperclip,
+  FaPhotoVideo,
+  FaPlay,
+  FaRegFile,
+  FaRegFolder,
+  FaRegImage,
+  FaTrash,
+} from 'react-icons/fa';
+import { Column, UseSortByColumnOptions } from 'react-table';
+import { Col, Row, Show } from 'tno-core';
+import { isImageFile, isVideoOrAudioFile } from 'utils';
+
+interface IColumnOptions {
+  isStorage?: boolean;
+  onSelect?: (item: IItemModel) => void;
+  onDelete?: (item: IItemModel) => void;
+  onPlay?: (item: IItemModel) => void;
+  onDownload?: (item: IItemModel) => void;
+  onAttach?: (item: IItemModel) => void;
+}
+
+/** columns located within file for state manipulation */
+export const fileExplorerColumns = (
```

Comment:
Need to find a way to pass this down from a parent component instead of hardcoding it.